Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/VehicleInspection.yaml

```diff
@@ -0,0 +1,24 @@
+imports:
+  Vehicle: Domain.Types.Vehicle
+
+VehicleInspection:
+  tableName: vehicle_inspection
```

Comment:
I don't think this info related with inspection
Maybe `vehicle_property` or `vehicle_info`?